### ISSUE TITLE
Taint analysis false negative on property access

### ISSUE THREAD

**Issue by JarLob** [dotnet/roslyn-analyzers#4065]
The following sample doesn't generate a warning. Tested on Master. Since it involves tainting a member value I'm not sure if this is by design or a bug.

```cs
using System.Data.SqlClient;

namespace sample
{
    public class TestInput
    {
        public void Run(string input)
        {
            Test a = new Test(input);
            SqlCommand c = a.Command;
        }
    }

    public class Test
    {
        private string sql;

        public Test(string s)
        {
            sql = s;
        }

        public SqlCommand Command
        {
            get
            {
                return new SqlCommand(sql);
            }
        }
    }
}
```
